usecases/traverser/traverser_explore_concepts.go

```diff
@@ -28,7 +28,7 @@ func (t *Traverser) Explore(ctx context.Context,
 		params.Limit = 20
 	}
 
-	err := t.authorizer.Authorize(principal, authorization.READ, authorization.CollectionsMetadata()...)
+	err := t.authorizer.Authorize(principal, authorization.READ, authorization.CollectionsData()...)
```

Comment:
Shouldn' it be `authorization.Collections()`? To check metadata and data? Or are you doing the metadata check somewhere else?